cypress/integration/work-order/index/filter.spec.js

```diff
@@ -464,6 +464,88 @@ describe('Filter work orders', () => {
       cy.get('[data-ref=10000032]')
     })
 
+    it('Clears filters one by one', () => {
+      cy.wait(['@filters', '@workOrders'])
+      //SElect Status filters
+      cy.get('.govuk-checkboxes').find('[name="StatusCode.90"]').check()
+      cy.get('.govuk-checkboxes').find('[name="StatusCode.80"]').check()
+      //Select Priority filters
+      cy.get('.govuk-checkboxes').find('[name="Priorities.2"]').check()
+
+      cy.get('[data-testid=apply-filters]').contains('Apply filters').click()
+
+      cy.wait([
+        '@filters',
+        '@workOrdersInProgressVariationPendingApprovalEmergency',
+      ])
+
+      cy.get('.selected-filters').within(() => {
+        cy.get('#selected-filters-Status').within(() => {
+          cy.contains('Variation Pending Approval')
+          cy.contains('In Progress')
+        })
+        cy.get('#selected-filters-Priority').within(() => {
+          cy.contains('Emergency')
+        })
+      })
+
+      //Removing filters manually one by one
+
+      //Removing Variation Pending Approval
+      cy.get('.selected-filters').within(() => {
+        cy.get('#selected-filters-Status').within(() => {
+          cy.contains('li', 'Variation Pending Approval').within(() => {
+            cy.get('button')
+              .should(
+                'have.attr',
+                'aria-label',
+                'Remove Variation Pending Approval filter'
+              )
+              .click()
```

Comment:
When we remove filters in this test, do any of them result in real API calls? An easy way to check is to turn off your connection, or to look at your logs - real API requests should show `Starting API Request` or similar. If so we need to create stubbed responses with `wait` calls, in the same way as for other requests in this file.

Reply:
We shouldn't need to bother with actually returning any work orders for these additional stubs - we're not asserting anything on the data that's returned. We can just base them on this, replacing the path and name as appropriate.

```
    cy.intercept(
      {
        path:
          '/api/workOrders/?PageSize=10&PageNumber=1&StatusCode=30&IncludeHistorical=false',
      },
      { body: [] }
    ).as('workOrdersCancelled')
```